lib/libraries/c3/extensions/tooltip-contents.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (d, defaultTitleFormat, defaultValueFormat, color) {
+  var $$ = this, config = $$.config,
+      titleFormat = config.tooltip_format_title || defaultTitleFormat,
```

Comment:
We should discuss what format we want to norm on for variable declarations. I'm cool with a single `var` statement, but I think we should keep it to one variable per line. Having 2 on the top line, then one per line for 3, then 6 per line is tough to read and easy to miss a variable when scanning code.